app/main.py

```diff
@@ -1 +1,12 @@
-# write your code here
+def copy_file(command: str) -> None:
+    if len(command.split()) < 3:
+        pass
```

Comment:
better to raise some informative error, so that it is clear to the user if something goes wrong